packages/patternfly-4/content/design-guidelines/styles/colors-for-charts.md

```diff
@@ -520,7 +520,7 @@ Colors for charts includes a recommended set of colors proposed for bar charts,
 ## The system
 Although the system may seem complex, it is completely customizable. The idea is that you can put together your desired colors and amount of colors and the system will still sort them correctly. Here’s how the system works:
 
-<img src={theSystem} />
+<img alt ="chart color system example" src={theSystem} />
```

Comment:
@lareplog can you make this: `alt="chart color system example"` and same for below, I think you only took out one of the spaces.

Reply:
oh my bad!! totally missed that space